advanced/security.md

```diff
@@ -2,46 +2,141 @@
 layout: default
 title: Security
 title_nav: Security
-description_short: A statement on security.
-description: A statement on security.
-keywords: security xss scripting vulnerability hack hacker
+description_short: Security information for TinyMCE.
+description: Information on reporting security issues, what TinyMCE does to protect users, and what you can do to protect your users.
+keywords: security xss scripting vulnerability hack hacker csp mitigation protection protect
 ---
 
-## Q: Is TinyMCE protected against XSS vulnerabilities?
+> **Note**: The following is _general_ security advice that may be relevant to website or application using {{site.productname}}.
 
-{{site.productname}} filters out some of the more common XSS content like scripts from the content since it is common that the editor is used in a single page application. For additional security, consider passing it through server-side filters like [HTMLPurifier](http://htmlpurifier.org/).
+## Overview
 
-## Q: How do I setup Content Security Policy (CSP) with TinyMCE?
+- [Reporting TinyMCE security issues](#reportingtinymcesecurityissues)
+- [What we do to maintain TinyMCEs' security](#whatwedotomaintaintinymcessecurity)
+
+  - [Scripts and XSS vulnerabilities](#scriptsandxssvulnerabilities)
+  - [Keeping dependencies up-to-date](#keepingdependenciesup-to-date)
+  - [Configuring Content Security Policy (CSP) for TinyMCE](#configuringcontentsecuritypolicycspfortinymce)
+- [Potential security risks and mitigation measures](#potentialsecurityrisksandmitigationmeasures)
+
+  - [Cross-Site Scripting (XSS)](#cross-sitescriptingxss)
+  - [Injection](#injection)
+  - [Insecure Transmission and Storage of data](#insecuretransmissionandstorageofdata)
+  - [Broken Authentication and Session Management](#brokenauthenticationandsessionmanagement)
+  - [Using Components with Known Vulnerabilities](#usingcomponentswithknownvulnerabilities)
+  - [Further security information](#furthersecurityinformation)
+
+## Reporting TinyMCE security issues
+
+{{site.companyname}} values the work of security researchers in improving the security of technology products worldwide. We welcome researchers who wish to responsibly disclose vulnerabilities in our products or systems. Note that we do not offer any “bug bounty” program or any form of payment for disclosed vulnerabilities. If you would like to report a vulnerability, please email `infosec@tiny.cloud`.
```

Comment:
Maybe `[infosec@tiny.cloud](mailto:infosec@tiny.cloud)` so that it's a link instead of plain text?

Reply:
I considered this, I decided not to link the email address to avoid automated spambots (somewhat). it is a security page after-all. I'm happy to link it if desired.